Ray[] rays
bool ignore_unknown_voxels
float32 max_range
---
float32 elapsed_seconds
int32 num_hits_occupied
int32 num_hits_free
int32 num_hits_unknown
float32 expected_reward
sensor_msgs/PointCloud2 point_cloud
